### PR TITLE
Removes briefcase of cash from the smuggler satchel loot pool

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Removes the briefcase of cash from the smuggler satchel pool and replaces it with a single 5000 credit bill, as were the contents of the briefcase.

## How This Contributes To The Skyrat Roleplay Experience

Having a briefcase inside a storage item is an oversight and not intended behaviour - if you take it out, you can't put it back in, and it can fit normal-sized (storage) items (like boxes, or BSRPEDs).
Fitting a box, in a briefcase, in a satchel, in a bag (and in a bag of holding, if you have one), is not how bag size and slots are meant to work.

## Changelog
:cl:
fix: The briefcase of cash has been replaced with cash in the smuggler's satchel loot pool.
/:cl:

### CODE REVIEW

**Comment by nevimer** [comment]
this ruins antagingg!! i cant storage my 50 large items in my bag within my bags bag's backpage in my storage implant!! evil!! /s

**Comment by Elli-Skala** [comment]
Wait tg doesn't have this problem also?

**Comment by tf-4** [comment]
> Wait tg doesn't have this problem also?

no it's a modular change

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -331,7 +331,7 @@
...
+	var/contraband_list = list(/obj/item/storage/bag/ammo=4, /obj/item/storage/belt/utility/syndicate=1, /obj/item/storage/toolbox/syndicate=7, /obj/item/card/id/advanced/chameleon=6, /obj/item/stack/spacecash/c5000=3, /obj/item/stack/telecrystal=2, /obj/item/storage/belt/military=12, /obj/item/storage/pill_bottle/aranesp=11, /obj/item/storage/pill_bottle/happy=12, /obj/item/storage/pill_bottle/stimulant=9, /obj/item/storage/pill_bottle/lsd=10, /obj/item/storage/fancy/cigarettes/cigpack_syndicate=8, /obj/item/storage/fancy/cigarettes/cigpack_shadyjims=10, /obj/item/reagent_containers/food/drinks/bottle/absinthe=12, /obj/item/storage/box/fireworks/dangerous=11, /obj/item/food/grown/cannabis/white=9, /obj/item/food/grown/cannabis=13, /obj/item/food/grown/cannabis/rainbow=8, /obj/item/food/grown/mushroom/libertycap=11, /obj/item/clothing/mask/gas/syndicate=10, /obj/item/vending_refill/donksoft=13, /obj/item/ammo_box/foambox/riot=11, /obj/item/soap/syndie=7)
```
Comment (GoldenAlpharex) [inline code/game/objects/items/storage/backpack.dm]: Can we make this list a bit more human-readable, while you're at it?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
One tiny thing, but otherwise, good change.

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me.

**Pull request merged**